flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/runtime/harness/GroupAggregateHarnessTest.scala

```diff
@@ -185,7 +191,7 @@ class GroupAggregateHarnessTest extends HarnessTestBase {
   @Test
   def testDistinctAggregateWithRetract(): Unit = {
     val env = StreamExecutionEnvironment.getExecutionEnvironment
-    val tEnv = StreamTableEnvironment.create(env)
+    val tEnv = StreamTableEnvironment.create(env, tableConfig)
```

Comment:
don't use deprecated constructor

Reply:
Unfortunately this is the only way as of now to pass the custom TableConfig, which we have to use because of the incompatible values used in these tests (see comment below)